homeassistant/helpers/entity_platform.py

```diff
@@ -62,6 +63,41 @@
 _LOGGER = getLogger(__name__)
 
 
+class Platform(StrEnum):
```

Comment:
If this will be the canonical list of entity integrations with platform support, that's great!

Reply:
Yes, that is the idea :)